### PR TITLE
feat(platform,platform-server): make project public to anyone

### PR DESCRIPTION
**Pull request by congjiujiu**
related to #65

截屏2022-09-14 17 23 41

截屏2022-09-14 14 51 51

### CODE REVIEW

**Comment by forehalo** [comment]
based on #54, make a grouped danger zone like github does
image

**Comment by EYHN** [comment]
Is there any permission issue? Users should only be able to visit the public project, and can't make any change.

**Comment by congjiujiu** [comment]
> based on #54, make a grouped danger zone like github does image

waiting for #54 to be merged

**Comment by congjiujiu** [comment]
> Is there any permission issue? Users should only be able to visit the public project, and can't make any change.

I will hide `lab -> take a snapshot` button and the `setting` route

**Comment by EYHN** [comment]
We should show some public projects on the home page for people to explore our features.

**Comment by congjiujiu** [comment]
> We should show some public projects on the home page for people to explore our features.

Do you mean the not logged in users

File: packages/schema/src/server-schema.gql [hunk 1]
```diff
@@ -248,6 +251,12 @@ enum GitHost {
   Gitlab
 }
 
+enum Permission {
+  All
```
Comment (forehalo) [inline packages/schema/src/server-schema.gql:255]: Remove `All` variant, it's the same as `Admin`

Comment (congjiujiu) [reply packages/schema/src/server-schema.gql]: Ok

**Review by forehalo — APPROVED** [review]

**Pull request merged**